Assets/MixedRealityToolkit-Preview/CameraCapture/CameraCaptureUWP.cs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#if WINDOWS_UWP
+using System;
+using System.Linq;
+using Unity.Collections;
+using UnityEngine;
+
+using System.Runtime.InteropServices;
+using UnityEngine.XR.WSA.WebCam;
+using Windows.Media.Devices;
+
+namespace Microsoft.MixedReality.Toolkit.Preview.CameraCapture
+{
+	public class CameraCaptureUWP : ICameraCapture
+	{
+		#region Fields
+		private PhotoCapture     camera      = null;
+		private CameraParameters cameraParams;
+		private CameraResolution resolution  = null;
+		private Texture2D        cacheTex    = null;
+		private Texture2D        resizedTex  = null;
+		private bool             isReady     = false;
+		private float            fieldOfView = 45;
+
+		/// <summary>
+		/// Is the camera completely initialized and ready to begin taking pictures?
+		/// </summary>
+		public bool  IsReady           { get { return isReady; } }
```

Comment:
formatting feels a bit odd here... not a merge blocker, but if you change this file for other reasons.... ;)